cpplint.py

```diff
@@ -6447,6 +6533,9 @@ def ProcessFileData(filename, file_extension, lines, error,
                 include_state, function_state, nesting_state, error,
                 extra_check_functions)
     FlagCxxHeaders(filename, clean_lines, line, error)
+  if _error_suppressions.HasOpenBlock():
+    error(filename, _error_suppressions.GetOpenBlockStart(), 'readability/nolint', 5,
```

Comment:
This should have a confidence level of 1 as it has legitimate usecases, such as as a NOLINTFILE.

Reply:
If you want to add NOLINTFILE that should probably be a different tag. Since this is linting for cpp which has explicit begin and ends to blocks I would argue that the end block should be required just like a `}` is required for blocks in cpp code.

Reply:
IMO that would just add another extra chore. } is just one character, while NOLINTEND is a lot more. Anyways this is not really a blocker as I also plan to reevaluate verbosities when I get the time, but I really need to know about https://github.com/cpplint/cpplint/pull/213#discussion_r1503333345 before this gets merged.

Also, I'd appreciate it if you fixed the lint errors.